virttest/utils_libvirt/libvirt_network.py

```diff
@@ -81,3 +83,62 @@ def check_established(params):
 
     cmdRes = remote.run_remote_cmd(cmd, params)
     libvirt.check_result(cmdRes, expected_match=exp_msg)
+
+
+def modify_network_xml(net_dict, testnet_xml):
+    """
+    modify the network's xml
+
+    :param testnet_xml: the network xml object to be modified
```

Comment:
doc string order doesn't match method orders

Reply:
And 1 blank line needed between description and params

Reply:
updated, Thank you!